build.gradle

```diff
@@ -151,27 +162,14 @@ subprojects {
 
         // The ErrorProne plugin defaults to the latest, which would break our
         // build if error prone releases a new version with a new check
-        errorprone "com.google.errorprone:error_prone_core:${errorProneVersion}"
-        errorproneJavac "com.google.errorprone:javac:${errorProneJavacVersion}"
-
-        if (JavaVersion.current().isJava9() || JavaVersion.current().isJava10() || JavaVersion.current().isJava11()) {
-            // Workaround for @javax.annotation.Generated
-            // see: https://github.com/grpc/grpc-java/issues/3633
-            compile("javax.annotation:javax.annotation-api:1.3.2")
```

Comment:
I think the PR build is broken because this bit needs to be here....

Reply:
Fixed, thanks :)